src/arduino/index.ts

```diff
@@ -132,12 +133,12 @@ export class ArduinoVS {
             status));
 
         spawn.stderr.on('data', data => {
-            let error:string = String.fromCharCode.apply(null, data);
+            let error:string = data.toString();
```

Comment:
this is my original design by use toString, it will success by most of times, but sometimes it will simply failed to convert to string. 

the only true way to convert this data is by use String.fromCharCode.apply, this will always success.